app/_components/header.tsx

```diff
@@ -10,49 +10,50 @@ import styles from "../_styles/scss/Header.module.scss";
 import clsx from "clsx";
 import Modal from "./modal";
 import { IoIosArrowDown } from "react-icons/io";
-import Button from '@mui/material/Button';
-import ModalComponent from '../_components/modal'; // Проверьте путь к вашему компоненту
+import Button from "@mui/material/Button";
+import ModalComponent from "../_components/modal"; // Проверьте путь к вашему компоненту
 import { IoIosArrowUp } from "react-icons/io";
 
-
 function Header() {
   const [open, setOpen] = useState(false);
 
-
-  // const handleOpen = () => setOpen(true);
-  // const handleClose = () => setOpen(false);
   const toggleMenu = () => {
-    setOpen(!open)
-  }
+    setOpen(!open);
+  };
 
   return (
-    <div className={styles.headerContainer}>
-      <div className={styles.headerLogoWrapper}>
-        <Link className={styles.headerLogo} href={"/"}>
-          <Image src={`/logo.png`} alt="logo" width={55} height={55} />
-          <span className={clsx(styles.headerLogoText, nunito.className)}>
-            Toloka
-          </span>
-        </Link>
-      </div>
-      <div>
-        <NavLinks />
-      </div>
-      <div className={styles.headerAccount}>
-        <Link className={scss.btnLogin} href="/">
-          УВІЙТИ
-        </Link>
+    <div>
+      <div className={styles.headerContainer}>
+        <div className={styles.headerLogoWrapper}>
+          <Link className={styles.headerLogo} href={"/"}>
+            <Image src={`/logo.png`} alt="logo" width={55} height={55} />
```

Comment:
no need to use ` - if no variables in the string 
please replace with ""
```suggestion 
            <Image src="/logo.png" alt="logo" width={55} height={55} />